src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -100,7 +100,7 @@ function AxisSystem(props: Props): JSX.Element {
             />
           </svg>
         </div>
-        {showGrid && (
+        {abscissaConfig.showGrid && ordinateConfig.showGrid && (
```

Comment:
We still need `abscissaConfig` and `ordinateConfig` in this component because of the grid. We could put the grid in its own component at some point.

Reply:
And use the context directly in the grid component. I agree.